### PR TITLE
Edited for wordchoice, contractions

### PR DESCRIPTION
**Pull request by Git-Good**
## Description

Edited README for word choice and usage of contractions.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ Premium and Open Source dashboard template with responsive and high quality UI.
...
+We created this admin panel for everyone who wants to create any templates based on our pre-made components. Our mission is to deliver a user-friendly, clear and easy administration panel, that can be used by both, simple websites and sophisticated systems. The only requirement is a basic HTML and CSS knowledge—as a reward, you'll be able to manage and visualize different types of data in the easiest possible way!
 
 * **Responsive:** With support for mobile, tablet and desktop it doesn’t matter what device you’re using. Tabler is responsive in all major browsers.
-* **Cross Browser:** Our theme works perfectly with: Latest Chrome, Firefox+, Latest Safari, Opera, Internet Explorer 10+ and mobile browsers. We’re working hard to support them.
+* **Cross Browser:** Our theme works perfectly with: Latest Chrome, Firefox+, Latest Safari, Opera, Internet Explorer 10+ and mobile browsers. We working hard to continuously support them.
```
Comment (beeman) [inline README.md]: I guess it should be `We are working hard`

**Comment by Git-Good** [comment]
Yes, edited.

**Comment by codecalm** [comment]
@Git-Good could you resolve the conflicts with readme file? Thanks for contributing!

**Comment by Git-Good** [comment]
Resolved the conflict.

**Comment by beeman** [comment]
Thanks for the contribution @Git-Good

**Pull request merged**